helix-term/src/commands.rs

```diff
@@ -291,6 +291,7 @@ impl Command {
         hsplit, "Horizontal bottom split",
         vsplit, "Vertical right split",
         wclose, "Close window",
+        wclose_force, "Close window discarding unsaved changes",
```

Comment:
I would drop this, we already have multiple ways of (force) quitting the editor and this is mainly for split manipulation.